code/modules/mob/living/silicon/robot/robot_mob.dm

```diff
@@ -42,81 +43,113 @@ GLOBAL_LIST_INIT(robot_verbs_default, list(
 	var/obj/item/stock_parts/cell/cell = null
 	var/obj/machinery/camera/camera = null
 
-	// Components are basically robot organs.
+	/// Components are basically robot organs.
 	var/list/components = list()
 
-	var/obj/item/robot_parts/robot_suit/robot_suit = null //Used for deconstruction to remember what the borg was constructed out of..
+	/// Used for deconstruction to remember what the robot was constructed out of.
+	var/obj/item/robot_parts/robot_suit/robot_suit = null
+	/// Used for deconstruction to remember what the robot was constructed out of.
 	var/obj/item/mmi/mmi = null
 
 	var/obj/item/pda/silicon/robot/rbPDA = null
 
 	var/datum/wires/robot/wires = null
 
+	/// Is the robot's maintenance panel open?
 	var/opened = FALSE
 	var/custom_panel = null
```

Comment:
What does this var do?

Reply:
> What does this var do?

Does the robot have a custom sprite for an open maintenance panel?

Also for some reason it's used for handling making eyes display on Cricket even though custom_eye_names handles that. I think. Maybe.